.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: lts/gallium
+          node-version: '18.x'
```

Comment:
There are a few other spots in this file that will need to be changed from `node-version: lts/gallium` to `node-version: '18.x'`